### ISSUE TITLE
distributive checks without associated service

### ISSUE THREAD

**Issue by ryane** [CiscoCloud/microservices-infrastructure#1066]
I noticed this with the vault distributive check but it seems to apply to most of the `distributive-*-check.json` consul health checks. Since the check is not associated with a service, when it fails, many other services are marked unhealthy as well. In the below screenshot, only the vault health check is failing but, at a glance, it looks like many services are having problems.

![consul-vault-check-no-service](https://cloud.githubusercontent.com/assets/7346/12720347/5aef010a-c8c8-11e5-994f-f6f421a6aac4.png)

If you bind the check to a vault service, it is a little easier to understand what is going wrong. For example, if you change `distributive-vault-check.json` to this:

```
{
  "service": {
    "name": "vault",
    "check": {
      "id": "distributive-vault-checks",
      "name": "Distributive vault Health Checks",
      "script": "/bin/distributive -f /etc/distributive.d/vault.json -d '' --verbosity=info",
      "interval": "1m",
      "timeout": "30s"
    }
  }
}
```

You end up with this:

![consul-vault-check-service](https://cloud.githubusercontent.com/assets/7346/12720353/6dc57d54-c8c8-11e5-8b3f-65bd1a38832e.png)

Is there a reason for the current setup or should we change the distributive health checks to be associated with services?

**Comment by stevendborrelli** [comment]
I agree with this, there are a lot of subtle interactions in setting up the health checks that we need to be aware of:

- health check knocking node offline in DNS
- failing health check on collectd taking the service it is monitoring offline

I'd also like to get rid of the distributive-naming. What do folks think of making it more generic:
```
service
service-config
service-logging
```

**Comment by siddharthist** [comment]
@stevendborrelli Agreed. @ryane I remember that we had some problems when checks were associated with specific services because they were named automatically and potentially confusingly. This might not be the case anymore, though.
